app/external_doc.rb

```diff
@@ -3,44 +3,6 @@
 require_relative "./string_to_id"
 
 class ExternalDoc
-  def self.fetch(repository:, path:)
```

Comment:
I think it might still be useful to keep this method around because it is much more convenient than doing `HTTP.get` in all places, but I don't feel too strongly about it.

Reply:
Thanks for the review, Tom. I don't feel hugely strongly about it either, but it's only being used [in one place at the moment](https://github.com/alphagov/govuk-developer-docs/blob/master/source%2Fmanual%2Fupdating-brexit-transition-checker.html.md.erb), and that will be removed shortly when we enable pulling in all repos' `docs/` folders anyway. So I suggest we leave it as removed, and add it back in later if there is a need.